[2025-02-13T07:42:03.047+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T07:42:03.069+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_data_dag.dbt_run manual__2025-02-13T07:42:01.137040+00:00 [queued]>
[2025-02-13T07:42:03.081+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_data_dag.dbt_run manual__2025-02-13T07:42:01.137040+00:00 [queued]>
[2025-02-13T07:42:03.082+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-13T07:42:03.098+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run> on 2025-02-13 07:42:01.137040+00:00
[2025-02-13T07:42:03.105+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4890) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T07:42:03.105+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'transform_data_dag', 'dbt_run', 'manual__2025-02-13T07:42:01.137040+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/transform_data_dag.py', '--cfg-path', '/tmp/tmp_dcttn7z']
[2025-02-13T07:42:03.107+0000] {standard_task_runner.py:72} INFO - Started process 4891 to run task
[2025-02-13T07:42:03.108+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask dbt_run
[2025-02-13T07:42:03.127+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-13T07:42:03.161+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_data_dag.dbt_run manual__2025-02-13T07:42:01.137040+00:00 [running]> on host 89bb8c6379b5
[2025-02-13T07:42:03.220+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/log/secrets_masker.py:84 DeprecationWarning: The sensitive_variable_fields option in [admin] has been moved to the sensitive_var_conn_names option in [core] - the old setting has been used, but please update your config.
[2025-02-13T07:42:03.252+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='transform_data_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T07:42:01.137040+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T07:42:01.137040+00:00'
[2025-02-13T07:42:03.253+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T07:42:03.270+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-13T07:42:03.271+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/dags/dbt/***_project && dbt run']
[2025-02-13T07:42:03.279+0000] {subprocess.py:99} INFO - Output:
[2025-02-13T07:42:05.647+0000] {subprocess.py:106} INFO - [0m07:42:05  Running with dbt=1.8.7
[2025-02-13T07:42:05.965+0000] {subprocess.py:106} INFO - [0m07:42:05  Registered adapter: ***=1.9.0
[2025-02-13T07:42:06.737+0000] {subprocess.py:106} INFO - [0m07:42:06  [[33mWARNING[0m]: Deprecated functionality
[2025-02-13T07:42:06.738+0000] {subprocess.py:106} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2025-02-13T07:42:06.738+0000] {subprocess.py:106} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2025-02-13T07:42:06.739+0000] {subprocess.py:106} INFO - information.
[2025-02-13T07:42:07.195+0000] {subprocess.py:106} INFO - [0m07:42:07  Found 6 models, 3 data tests, 3 sources, 429 macros
[2025-02-13T07:42:07.199+0000] {subprocess.py:106} INFO - [0m07:42:07
[2025-02-13T07:42:07.327+0000] {subprocess.py:106} INFO - [0m07:42:07  Concurrency: 1 threads (target='dev')
[2025-02-13T07:42:07.328+0000] {subprocess.py:106} INFO - [0m07:42:07
[2025-02-13T07:42:07.402+0000] {subprocess.py:106} INFO - [0m07:42:07  1 of 3 START sql incremental model model.order_items ........................... [RUN]
[2025-02-13T07:42:07.696+0000] {subprocess.py:106} INFO - [0m07:42:07  1 of 3 OK created sql incremental model model.order_items ...................... [[32mINSERT 0 0[0m in 0.29s]
[2025-02-13T07:42:07.699+0000] {subprocess.py:106} INFO - [0m07:42:07  2 of 3 START sql incremental model model.orders ................................ [RUN]
[2025-02-13T07:42:07.829+0000] {subprocess.py:106} INFO - [0m07:42:07  2 of 3 OK created sql incremental model model.orders ........................... [[32mINSERT 0 0[0m in 0.13s]
[2025-02-13T07:42:07.832+0000] {subprocess.py:106} INFO - [0m07:42:07  3 of 3 START sql incremental model model.products .............................. [RUN]
[2025-02-13T07:42:07.993+0000] {subprocess.py:106} INFO - [0m07:42:07  3 of 3 OK created sql incremental model model.products ......................... [[32mINSERT 0 0[0m in 0.16s]
[2025-02-13T07:42:08.010+0000] {subprocess.py:106} INFO - [0m07:42:08
[2025-02-13T07:42:08.011+0000] {subprocess.py:106} INFO - [0m07:42:08  Finished running 3 incremental models in 0 hours 0 minutes and 0.81 seconds (0.81s).
[2025-02-13T07:42:08.149+0000] {subprocess.py:106} INFO - [0m07:42:08
[2025-02-13T07:42:08.149+0000] {subprocess.py:106} INFO - [0m07:42:08  [32mCompleted successfully[0m
[2025-02-13T07:42:08.150+0000] {subprocess.py:106} INFO - [0m07:42:08
[2025-02-13T07:42:08.151+0000] {subprocess.py:106} INFO - [0m07:42:08  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-02-13T07:42:08.915+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-13T07:42:08.945+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T07:42:08.946+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=transform_data_dag, task_id=dbt_run, run_id=manual__2025-02-13T07:42:01.137040+00:00, execution_date=20250213T074201, start_date=20250213T074203, end_date=20250213T074208
[2025-02-13T07:42:09.028+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T07:42:09.046+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-13T07:42:09.049+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
